module/app/app.go

```diff
@@ -582,7 +598,12 @@ func NewGravityApp(
 		bankKeeper,
 		stakingKeeper,
 		govRouter,
+		app.MsgServiceRouter(),
+		govConfig,
 	)
+	govKeeper = *govKeeper.SetHooks(govtypes.NewMultiGovHooks(
+	// Register any governance hooks here
```

Comment:
do we need any hooks for our custom governance proposals?

Reply:
These are new hooks that have been added. Our proposals work off of the 0.45 method so we don't need to put anything here. This does open up some flexible new options though.